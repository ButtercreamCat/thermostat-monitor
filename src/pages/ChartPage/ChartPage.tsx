import React, { Component } from "react";
import TemperatureChart from "../../components/TemperatureChart/TemperatureChart";

export default class ChartPage extends Component {
  render() {
    return (
      <div className="unitInfo_content">
        <TemperatureChart></TemperatureChart>
      </div>
    );
  }
}
