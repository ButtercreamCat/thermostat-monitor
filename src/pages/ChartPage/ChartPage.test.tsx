import React from "react";
import { shallow } from "enzyme";
import ChartPage from "./ChartPage";

test("should render chart page correctly", () => {
  const wrapper = shallow(<ChartPage />);
  expect(wrapper).toMatchSnapshot();
});
