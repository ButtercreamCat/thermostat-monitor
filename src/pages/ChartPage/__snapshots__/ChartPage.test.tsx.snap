// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render chart page correctly 1`] = `
<div
  className="unitInfo_content"
>
  <TemperatureChart />
</div>
`;
